Framework/Algorithms/src/Q1DWeighted.cpp

```diff
@@ -85,63 +87,46 @@ void Q1DWeighted::init() {
 
   declareProperty("AsymmetricWedges", false,
                   "Choose to produce the results for asymmetric wedges.");
+
+  declareProperty("AccountForGravity", false,
+                  "Take the nominal gravity drop into account.");
 }
 
 void Q1DWeighted::exec() {
```

Comment:
This function is extremely long. It could be broken into smaller more manageable pieces, I think.